canonical-webteam/global-nav
# Add some default styling

## Done
Add default styling to override the styling applied when used on a site.

## QA
- Check that the styles look ok
- Run `gulp build`
- Open index.html
- Check that the global-nav is the same
